server/src/db/LawyerSeeder.js

```diff
@@ -0,0 +1,25 @@
+import { Lawyer } from "../models/index.js";
+class LawyerSeeder {
+  static async seed() {
+    const lawyerData = [
+      {
+        name: "Jane",
+        id: "3",
```

Comment:
should not seed data with an `id` - the database will create the `id` value for us and will auto-increment the value anytime a new record is created